src/amber/cli/templates/app/src/assets/javascripts/main.js

```diff
@@ -1 +1,4 @@
+import 'jquery';
+import 'popper.js';
```

Comment:
I think bootstrap will import jquery and popper itself, so you don't have to include these explicitly.

Reply:
Easy, updated.